fv3config/_datastore.py

```diff
@@ -45,11 +49,15 @@ def get_orographic_forcing_directory(config):
     specified by a config dictionary.
     """
     resolution = get_resolution(config)
-    dirname = os.path.join(get_internal_cache_dir(), f"orographic_data/{resolution}")
-    if not os.path.isdir(dirname):
-        valid_options = os.listdir(
-            os.path.join(get_internal_cache_dir(), "orographic_data")
-        )
+    if 'orographic_forcing' not in config:
+        orographic_forcing_spec = 'default'
+    else:
+        orographic_forcing_spec = config['orographic_forcing']
+    parent_dirname = resolve_option(orographic_forcing_spec, OROGRAPHIC_FORCING_OPTIONS_DICT)
```

Comment:
`orographic_forcing_spec` isn't any more descriptive than `config.get('orographic_forcing', 'default')`, so I'd suggest replacing this with:
`parent_dirname = resolve_option(config.get('orographic_forcing', 'default'), OROGRAPHIC_FORCING_OPTIONS_DICT)`

Reply:
sure